StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/VerticalPaymentMethodListViewControllerTest.swift

```diff
@@ -74,7 +74,7 @@ final class VerticalPaymentMethodListViewControllerTest: XCTestCase {
             incentive: nil,
             delegate: self
         )
-        XCTAssertEqual(["SEPA Debit", "Card", "Apple Pay", "Link"], sut.rowButtons.map { $0.label.text })
+        XCTAssertEqual(["SEPA Debit", "Card", "Apple Pay", "Link"], sut.rowButtons.map { $0.text })
```

Comment:
`text` on RowButton is flagged as dead but used here, lmk what you think.

Reply:
Looks like `text` just contains the value that was passed as the initializer's `text` parameter. What's the intention?  Based on this use case, `text` should be a computed var that returns the label's text.